lms/templates/student_profile/learner_performance.html

```diff
@@ -0,0 +1,5 @@
+%if data['profile_statistics']:
+    ${data['profile_statistics'].content}
+%else :
+     <h1>No data ${data['profile_statistics']}</h1>
+% endif
```

Comment:
Please add a new line at the end of the file (here and in other cases, whenever missing)

Reply:
...for the sake of this: https://unix.stackexchange.com/a/18789

Reply:
resolved